diff -urNp truecrypt-6.1-source.orig/Common/Crypto.h truecrypt-6.1-source/Common/Crypto.h
--- truecrypt-6.1-source.orig/Common/Crypto.h	2008-11-11 23:12:53.000000000 +0300
+++ truecrypt-6.1-source/Common/Crypto.h	2008-11-11 23:14:33.000000000 +0300
@@ -29,6 +29,10 @@
 
 #include "Tcdefs.h"
 
+#ifndef BOOL
+#define BOOL BOOL1
+#endif
+
 #ifdef __cplusplus
 extern "C" {
 #endif
diff -urNp truecrypt-6.1-source.orig/Makefile truecrypt-6.1-source/Makefile
--- truecrypt-6.1-source.orig/Makefile	2008-11-11 23:12:53.000000000 +0300
+++ truecrypt-6.1-source/Makefile	2008-11-11 23:13:32.000000000 +0300
@@ -50,6 +50,7 @@ C_CXX_FLAGS += -I$(PKCS11_INC)
 endif
 
 C_CXX_FLAGS += -DBOOL=int -DFALSE=0 -DTRUE=1
+C_CXX_FLAGS += -DBOOL1=int
 ifeq "$(origin USE64BIT)" "command line"
 C_CXX_FLAGS += -D__int8=char -D__int16=short -D__int32=int '-D__int64=long'  # Tested in PlatformTest
 else
