diff -rupa kdevelop-3.5.5.orig/embedded/visualboyadvance/visualboyadvance_part.cpp kdevelop-3.5.5/embedded/visualboyadvance/visualboyadvance_part.cpp
--- kdevelop-3.5.5.orig/embedded/visualboyadvance/visualboyadvance_part.cpp	2011-03-01 01:51:38.000000000 +0300
+++ kdevelop-3.5.5/embedded/visualboyadvance/visualboyadvance_part.cpp	2011-03-01 02:28:00.000000000 +0300
@@ -63,7 +63,7 @@ void VisualBoyAdvancePart::slotExecute()
   program += prj->projectDirectory() + "/" + binary;
 
   if (KDevAppFrontend *appFrontend = extension<KDevAppFrontend>("KDevelop/AppFrontend"))
-      appFrontend->startAppCommand(QString::QString(), program, terminal);
+      appFrontend->startAppCommand(QString(), program, terminal);
 }
 
 void VisualBoyAdvancePart::projectConfigWidget(KDialogBase *dlg){
diff -rupa kdevelop-3.5.5.orig/languages/bash/bashsupport_part.cpp kdevelop-3.5.5/languages/bash/bashsupport_part.cpp
--- kdevelop-3.5.5.orig/languages/bash/bashsupport_part.cpp	2011-03-01 01:51:37.000000000 +0300
+++ kdevelop-3.5.5/languages/bash/bashsupport_part.cpp	2011-03-01 02:13:51.000000000 +0300
@@ -183,7 +183,7 @@ void BashSupportPart::startApplication(c
 {
 	kdDebug() << "starting application" << program << endl;
 	if (KDevAppFrontend *appFrontend = extension<KDevAppFrontend>("KDevelop/AppFrontend"))
-		appFrontend->startAppCommand(QString::QString(), program, TRUE);
+		appFrontend->startAppCommand(QString(), program, TRUE);
 }
 
 
diff -rupa kdevelop-3.5.5.orig/languages/csharp/csharpsupportpart.cpp kdevelop-3.5.5/languages/csharp/csharpsupportpart.cpp
--- kdevelop-3.5.5.orig/languages/csharp/csharpsupportpart.cpp	2011-03-01 01:51:37.000000000 +0300
+++ kdevelop-3.5.5/languages/csharp/csharpsupportpart.cpp	2011-03-01 02:17:09.000000000 +0300
@@ -194,7 +194,7 @@ void CSharpSupportPart::startApplication
 {
     bool inTerminal = DomUtil::readBoolEntry(*projectDom(), "/kdevcsharpsupport/run/terminal");
     if (KDevAppFrontend *appFrontend = extension<KDevAppFrontend>("KDevelop/AppFrontend"))
-        appFrontend->startAppCommand(QString::QString(), program, inTerminal);
+        appFrontend->startAppCommand(QString(), program, inTerminal);
 }
 
 
diff -rupa kdevelop-3.5.5.orig/languages/perl/perlparser.cpp kdevelop-3.5.5/languages/perl/perlparser.cpp
--- kdevelop-3.5.5.orig/languages/perl/perlparser.cpp	2011-03-01 01:51:37.000000000 +0300
+++ kdevelop-3.5.5/languages/perl/perlparser.cpp	2011-03-01 02:19:41.000000000 +0300
@@ -464,7 +464,7 @@ void perlparser::getPerlINC() {
             return;
         }
         array.setRawData(buffer, n);
-        result=QString::QString(array);
+        result=QString(array);
         array.resetRawData(buffer, n);
  }
  pclose(fd);
diff -rupa kdevelop-3.5.5.orig/languages/perl/perlsupportpart.cpp kdevelop-3.5.5/languages/perl/perlsupportpart.cpp
--- kdevelop-3.5.5.orig/languages/perl/perlsupportpart.cpp	2011-03-01 01:51:37.000000000 +0300
+++ kdevelop-3.5.5/languages/perl/perlsupportpart.cpp	2011-03-01 02:20:08.000000000 +0300
@@ -194,7 +194,7 @@ void PerlSupportPart::startApplication(c
 {
     bool inTerminal = DomUtil::readBoolEntry(*projectDom(), "/kdevperlsupport/run/terminal");
     if (KDevAppFrontend *appFrontend = extension<KDevAppFrontend>("KDevelop/AppFrontend"))
-        appFrontend->startAppCommand(QString::QString(), program, inTerminal);
+        appFrontend->startAppCommand(QString(), program, inTerminal);
 }
 
 
diff -rupa kdevelop-3.5.5.orig/languages/python/pythonsupportpart.cpp kdevelop-3.5.5/languages/python/pythonsupportpart.cpp
--- kdevelop-3.5.5.orig/languages/python/pythonsupportpart.cpp	2011-03-01 01:51:37.000000000 +0300
+++ kdevelop-3.5.5/languages/python/pythonsupportpart.cpp	2011-03-01 02:23:15.000000000 +0300
@@ -328,7 +328,7 @@ void PythonSupportPart::startApplication
 {
     bool inTerminal = DomUtil::readBoolEntry(*projectDom(), "/kdevpythonsupport/run/terminal");
     if (KDevAppFrontend *appFrontend = extension<KDevAppFrontend>("KDevelop/AppFrontend"))
-        appFrontend->startAppCommand(QString::QString(), program, inTerminal);
+        appFrontend->startAppCommand(QString(), program, inTerminal);
 }
 
 
diff -rupa kdevelop-3.5.5.orig/languages/ruby/rubysupport_part.cpp kdevelop-3.5.5/languages/ruby/rubysupport_part.cpp
--- kdevelop-3.5.5.orig/languages/ruby/rubysupport_part.cpp	2011-03-01 01:51:37.000000000 +0300
+++ kdevelop-3.5.5/languages/ruby/rubysupport_part.cpp	2011-03-01 02:25:01.000000000 +0300
@@ -614,7 +614,7 @@ QString RubySupportPart::characterCoding
 void RubySupportPart::startApplication(const QString &program) {
 	bool inTerminal = DomUtil::readBoolEntry(*projectDom(), "/kdevrubysupport/run/terminal");
     if (KDevAppFrontend *appFrontend = extension<KDevAppFrontend>("KDevelop/AppFrontend"))
-        appFrontend->startAppCommand(QString::QString(), program, inTerminal);
+        appFrontend->startAppCommand(QString(), program, inTerminal);
 }
 
 
diff -rupa kdevelop-3.5.5.orig/parts/tools/tools_part.cpp kdevelop-3.5.5/parts/tools/tools_part.cpp
--- kdevelop-3.5.5.orig/parts/tools/tools_part.cpp	2011-03-01 01:51:37.000000000 +0300
+++ kdevelop-3.5.5/parts/tools/tools_part.cpp	2011-03-01 01:52:15.000000000 +0300
@@ -154,7 +154,7 @@ void ToolsPart::startCommand(QString cmd
     if (captured)
     {
        if (KDevAppFrontend *appFrontend = extension<KDevAppFrontend>("KDevelop/AppFrontend"))
-            appFrontend->startAppCommand(QString::QString(), cmdline, false);
+            appFrontend->startAppCommand(QString(), cmdline, false);
     }
     else 
     {
